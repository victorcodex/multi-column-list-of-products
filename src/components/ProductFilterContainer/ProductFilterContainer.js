import styled from "styled-components";

const ProductFilterContainer = styled.section`
    display: flex;
    justify-content: center;
    align-items: center;
    column-gap: 2vh;
    margin-bottom: 3vh;
`;

export default ProductFilterContainer;