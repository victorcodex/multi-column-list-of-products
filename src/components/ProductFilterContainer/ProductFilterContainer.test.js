import React from 'react';
import { render } from '@testing-library/react';
import ProductFilterContainer from './ProductFilterContainer';

describe('ProductFilterContainer component', () => {
  it('should render filter container', () => {
    const { asFragment } = render(<ProductFilterContainer />);
    expect(asFragment()).toMatchSnapshot();
  });
});