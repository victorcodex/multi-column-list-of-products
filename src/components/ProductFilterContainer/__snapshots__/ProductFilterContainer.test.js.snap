// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ProductFilterContainer component should render filter container 1`] = `
<DocumentFragment>
  <section
    class="sc-bdnxRM DImr"
  />
</DocumentFragment>
`;
